scripts/microbe_stage/microbe_stage_hud.lua

```diff
@@ -63,7 +65,8 @@ function HudSystem:update(renderTime)
 
     self.hitpointsBar:progressbarSetProgress(playerMicrobe.microbe.hitpoints/playerMicrobe.microbe.maxHitpoints)
     self.hitpointsCountLabel:setText("".. math.floor(playerMicrobe.microbe.hitpoints))
-    
+    local playerSpecies = playerMicrobe:getSpeciesComponent()
+    --self.populationNumberLabel:setText("" .. math.floor(playerSpecies.currentPopulation))
```

Comment:
I probably need to read further down to answer this, but could we continue doing this? Adding a basic label shouldn't be too difficult